<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonExportTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>export to...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TqLineEdit" name="lineEditExportTo"/>
      </item>
      <item>
       <widget class="TqComboBox" name="comboBoxDSP"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="TqTreeView" name="treeViewImportedFiles"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="buttonProcessFiles">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>process files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TqComboBox" name="comboBoxRemoveFiles">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Audio_Files"/>
    <addaction name="actionImport_folder"/>
    <addaction name="separator"/>
    <addaction name="actionChoose_Export_Folder"/>
    <addaction name="actionShow_export_folder"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRemove_all_files"/>
    <addaction name="actionRemove_selected_files"/>
    <addaction name="actionRemove_finished_files"/>
    <addaction name="separator"/>
    <addaction name="actionProcess"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionImport_Audio_Files">
   <property name="text">
    <string>import audio files...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionChoose_Export_Folder">
   <property name="text">
    <string>choose export folder...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionShow_export_folder">
   <property name="text">
    <string>show export folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionRemove_selected_files">
   <property name="text">
    <string>remove selected</string>
   </property>
  </action>
  <action name="actionRemove_all_files">
   <property name="text">
    <string>remove all</string>
   </property>
  </action>
  <action name="actionRemove_finished_files">
   <property name="text">
    <string>remove finished</string>
   </property>
  </action>
  <action name="actionProcess">
   <property name="text">
    <string>process...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionImport_folder">
   <property name="text">
    <string>import folder...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TqTreeView</class>
   <extends>QTreeView</extends>
   <header>tqtreeview.h</header>
  </customwidget>
  <customwidget>
   <class>TqLineEdit</class>
   <extends>QLineEdit</extends>
   <header>tqlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>TqComboBox</class>
   <extends>QComboBox</extends>
   <header>tqcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
